packages/edit-site/src/components/sidebar-dataviews/index.js

```diff
@@ -14,24 +14,19 @@ const { useLocation } = unlock( routerPrivateApis );
 import DataViewItem from './dataview-item';
 import CustomDataViewsList from './custom-dataviews-list';
 
-const PATH_TO_TYPE = {
-	'/page': 'page',
-};
-
 export default function DataViewsSidebarContent() {
 	const {
-		params: { path, activeView = 'all', isCustom = 'false' },
+		params: { postType, activeView = 'all', isCustom = 'false' },
```

Comment:
I think we should unify between "activeView" in the pages page and "categoryId" in the patterns page.

Reply:
Maybe we should do that when we actually use views in patterns page to avoid creating extra confusion with nomenclature.

Reply:
Isn't it just the "default views" of patterns though, just like we have "default views" for pages (regardless of whether you can create views)

Reply:
Semantically yes, technically no. With nomenclature confusion I meant mostly towards devs.